dapps/assets/src/components/contacts/Contacts.vue

```diff
@@ -0,0 +1,175 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <div>
+    <div class="content pt-5">
+      <div class="d-flex flex-row justify-content-between align-items-center">
+        <div>
+          <h1 class="h4" style="margin:0">{{ '_assets.contacts.contacts-title' | translate }}</h1>
+        </div>
+        <div>
+          <evan-button
+            @click="filterByFavorites()"
+            class="filter-btn ml-3"
+            type="text-secondary"
+            icon="mdi mdi-star-outline"
+            iconPosition="left"
+            :class="{ 'active': filterBy.includes('favorite') }"
+            :label="$t('_assets.contacts.favorites')"
+          ></evan-button>
+          <evan-button
+            @click="filterByType('users')"
+            class="filter-btn ml-3"
+            type="text-secondary"
+            icon="mdi mdi-account-outline"
+            iconPosition="left"
+            :class="{ 'active': filter === 'users' }"
+            :label="$t('_assets.contacts.users')"
+          ></evan-button>
```

Comment:
I think this can all be written as self closing component.

`/>` instead of `></evan-button>`